crates/egui/src/atomics/atomic.rs

```diff
@@ -0,0 +1,119 @@
+use crate::{AtomicKind, SizedAtomic, Style, Ui};
+use emath::Vec2;
+use epaint::text::TextWrapMode;
+use epaint::Fonts;
+
+/// A low-level ui building block.
+///
+/// Implements [`From`] for [`String`], [`str`], [`crate::Image`] and much more for convenience.
+/// You can directly call the `a_*` methods on anything that implements `Into<Atomic>`.
+/// ```
+/// # use egui::{Image, emath::Vec2};
+/// use egui::AtomicExt;
+/// let string_atomic = "Hello".a_grow(true);
+/// let image_atomic = Image::new("some_image_url").a_size(Vec2::splat(20.0));
+/// ```
+#[derive(Clone, Debug)]
+pub struct Atomic<'a> {
```

Comment:
And this is a whopping 168 bytes 😱

Reply:
A lot of it is `WidgetText`, which is 112 bytes. `WidgetText` could use some slimming down at some point…